### DISCUSSION TITLE
Launch of a new site that user ABP - https://webinarcafe.com

### DISCUSSION THREAD

**Discussion by jasenf** [abpframework/abp#18732] [Show and tell]
Hi all -- 

Just wanted to say I recently launched a new website / side hustle I've been working on for over a year.  It uses abp.io so thought everyone may be interested in checking it out. 

It's a website directory and newsletter where organizations can promote upcoming webinars and individuals can get daily newsletters listing webinars on topics they decide they are interested in.

Hope you find it interesting! 😁

jasen
https://webinarcafe.com

**Comment by maliming** [comment]
Congratulations, it looks great.

**Comment by tarikozdemir** [comment]
Looks fantastic! Keep going...

**Comment by realLiangshiwei** [comment]
Congratulations, looks very professional.

**Comment by ITconsultants247** [comment]
This is really great.  Would you be able to share with us a few technical information?
Community or commercial abp framework?  Application type? Layered or not? Backend DB? Front-end UI?
you did a great job, regardless.  Please let us know if you decide to start blogging about your experience with this project, I would love to subscribe and follow your blog. Cheers 🥂

**Reply by jasenf** [reply]
Thanks for the kind words.  Commercial, custom theme obviously.  MVC/SQL.   I am considering launching a .NET / C# newsletter.  WebinarCafe hasn't really found much growth yet.

**Reply by Stroudwater** [reply]
whether WebinarCafe grows or not, it won't change the fact that you did a great job.  I have been using abp, commercial, for more than a year now for the company I work for, I can recognize the hard work and efforts you put in your site and that what matters.

**Comment by tarikozdemir** [comment]
I have an idea, why do not we create a success story for your project?

ABP Marketing Team Leader

**Reply by jasenf** [reply]
Would be happy to contribute / cross promote.  Been a huge fan of ABP for a looong time.  I recently sold my last SaaS (statuscast.com and have started building my next big product on abp.  I'm a 1 man coding shop right now though so unless I found some good abp developers that are affordable its going to take a bit.

anyway happy to discuss webinarcafe.com at any time.

**Reply by tarikozdemir** [reply]
I have sent you a message in Linkedin, please check.
